### PR TITLE
[bitnami/thanos] Add custom tsdb path

### PR DESCRIPTION
**Pull request by alecrajeev**
### Description of the change

Adding an option for a custom tsdb path.

### Benefits

This allows a user to set a tsdb path for a custom volume mount.

### Additional information

This makes it much easier to set an ephemeral volume for the tsdb.

* https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/thanos/Chart.yaml [hunk 1]
```diff
@@ -35,4 +35,4 @@ maintainers:
 name: thanos
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/thanos
-version: 15.2.1
+version: 15.2.2
```
Comment (dgomezleon) [inline bitnami/thanos/Chart.yaml:38]:
```suggestion
version: 15.3.0
```

**Review by dgomezleon — CHANGES_REQUESTED** [review]
Thank you for contributing. Please bump it as a minor version as it includes new values.

**Comment by alecrajeev** [comment]
> Thank you for contributing. Please bump it as a minor version as it includes new values.

good suggestion

File: bitnami/thanos/README.md [hunk 2]
```diff
@@ -1236,6 +1236,7 @@ You can enable this initContainer by setting `volumePermissions.enabled` to `tru
...
 | `receive.replicationFactor`                                 | Thanos Receive replication-factor                                                                                                                                                                                                 | `1`                      |
 | `receive.config`                                            | Receive Hashring configuration                                                                                                                                                                                                    | `[]`                     |
+| `receive.tsdbPath`                                          | path to the time series database                                                                                                                                                                                                  | `""`                     |
```
[the remark this answers is not in the document]
Comment (alecrajeev) [reply bitnami/thanos/README.md]: thank you for fixing this!

**Pull request merged**